.github/workflows/examples.yml

```diff
@@ -40,6 +41,15 @@ jobs:
         source activate test-environment
         pytest --verbose tests/examples --large
 
+    - name: Clean up conda environments
+      run: |
+        conda env list |                        # list (env name, env path) pairs
+        cut -d' ' -f1 |                         # extract env names
+        grep "^mlflow-[a-z0-9]\{32\}" |         # filter envs created by mlflow
+        xargs -L 1 conda remove --all --yes -n  # pass env names to conda remove
```

Comment:
Maybe this command should be put in a shell script so we can use it  when we want to clean up all conda environments created by `mlflow run` for some reason.

Reply:
Probably we should run something similar after the master tests workfow, as I think the large tests also create conda envs.

Reply:
Let me just chekc if the large tests really create conda envs.

Reply:
Looks like both small and large tests create some conda envs.

small: https://github.com/mlflow/mlflow/pull/3245/checks?check_run_id=967639307#step:5:9
large: https://github.com/mlflow/mlflow/pull/3245/checks?check_run_id=967639346#step:5:9

Reply:
If the number of conda envs were large, we'd need to.